# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../hostapd/install.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ruN hostapd-0.6.10-orig/hostapd/Makefile hostapd-0.6.10/hostapd/Makefile
--- hostapd-0.6.10-orig/hostapd/Makefile	2010-01-12 15:55:02.000000000 +0100
+++ hostapd-0.6.10/hostapd/Makefile	2010-08-04 17:40:45.320090875 +0200
@@ -1,3 +1,6 @@
+DESTDIR =
+prefix = /usr/local
+
 ifndef CC
 CC=gcc
 endif
@@ -577,7 +580,7 @@
 	fi
 
 install: all
-	for i in $(ALL); do cp $$i /usr/local/bin/$$i; done
+	for i in $(ALL); do cp $$i $(DESTDIR)/$(prefix)/bin/$$i; done
 
 hostapd: $(OBJS)
 	$(CC) -o hostapd $(OBJS) $(LIBS)
